### PR TITLE
Backport sandbox remote exception fix

### PR DESCRIPTION
**Pull request by tarruda**

### CODE REVIEW

File: lib/renderer/api/remote.js [hunk 1]
```diff
@@ -304,6 +304,10 @@ ipcRenderer.on('ELECTRON_RENDERER_RELEASE_CALLBACK', function (event, id) {
   callbacksRegistry.remove(id)
 })
 
+process.on('exit', () => {
+  ipcRenderer.sendSync('ELECTRON_BROWSER_CONTEXT_RELEASE')
```
Comment (kevinsawicki) [inline lib/renderer/api/remote.js]: Just to confirm, this is only needed for sanboxed renderer processes right? Or was this issue also affecting non-sandboxed renderers?

**Comment by tarruda** [comment]
It also affects `nativeWindowOpen` child windows created with
`window.open`. For version 1.6.x, it only affects sandbox.

**Comment by kevinsawicki** [comment]
> It also affects `nativeWindowOpen` child windows created with
`window.open`. For version 1.6.x, it only affects sandbox.

Great, thanks for clarifying 👍

**Comment by kevinsawicki** [comment]
And thanks for backporting this to 1.6

**Pull request merged**